docs/Data_Portal/Users_Guide/quick_start.md

```diff
@@ -11,23 +13,74 @@ __Step 1:__ Go to the Cohort Builder at the top left of the Data Portal.
 [![Animation of Browsing to the Cohort Builder](images/agifs/GotoCB-animation.gif)](images/agifs/GotoCB-animation.gif "Click to see the full image.")
 </details>
 
+---
+
 __Step 2:__ Use the filters in the Cohort Builder to filter down the full set of GDC cases to a subset you are interested in. Filter categories can be selected in the left panel.
 
 <details>
   <summary><i>Click to Expand/Collapse Animation</i></summary>
 [![Animation of Filtering Down Cohort](images/agifs/Filter-animation.gif)](images/agifs/Filter-animation.gif "Click to see the full image.")
 </details>
 
+---
+
 __Step 3:__ Save your cohort by clicking the "Save" icon in the cohort bar, choosing "Save", and naming your cohort when prompted.
 
 <details>
   <summary><i>Click to Expand/Collapse Animation</i></summary>
 [![Animation of Saving a Cohort](images/agifs/CohortSave-animation.gif)](images/agifs/CohortSave-animation.gif "Click to see the full image.")
 </details>
 
+---
+
 __Step 4:__ The cohort you created is now your active cohort.  Go to the Analysis Center at the top left of the Data Portal.  Choose the tool you would like to use from the list.  The analysis will apply to the data from your cohort.
 
 <details>
   <summary><i>Click to Expand/Collapse Animation</i></summary>
 [![Animation of Browsing to the Analysis Center](images/agifs/GotoAC-animation.gif)](images/agifs/GotoAC-animation.gif "Click to see the full image.")
 </details>
+
+## Downloading Files
+
+__Step 1:__ Have an active cohort selected.  If you do not have an active cohort, [create one](#building-and-analyzing-a-cohort).
+
+<details>
+  <summary><i>Click to Expand/Collapse Animation</i></summary>
+[![Animation of Selecting an Active Cohort](images/agifs/ChangeCohort-animation.gif)](images/agifs/ChangeCohort-animation.gif "Click to see the full image.")
+</details>
+
+---
+
+__Step 2:__ Go to the Repository tool from the Analysis Center or the top left of the portal.
+
+<details>
+  <summary><i>Click to Expand/Collapse Animation</i></summary>
+[![Animation of Browsing to the Repository](images/agifs/GoToRepo-animation.gif)](images/agifs/GoToRepo-animation.gif "Click to see the full image.")
+</details>
+
+---
+
+__Step 3:__ In the Repository tool, the files will only be those that are associated with the cases in your active cohort. Narrow down your file selection by filtering them using the panel on the left.
+
+<details>
+  <summary><i>Click to Expand/Collapse Animation</i></summary>
+[![Animation of Filtering the Repository](images/agifs/SelectFiltersRepo-animation.gif)](images/agifs/SelectFiltersRepo-animation.gif "Click to see the full image.")
+</details>
+
+---
+
+__Step 4:__ When you are done filtering.  Add the files to your cart.  Then go to the cart.
```

Comment:
Maybe a typo (?) - "When you are done filtering, add the files to your cart."

Reply:
100% a typo, will fix with next commit.